f8a_worker/workers/license.py

```diff
@@ -112,4 +111,9 @@ def execute(self, arguments):
             cache_path = ObjectCache.get_from_dict(arguments).get_extracted_source_tarball()
 
         result_data = self.run_scancode(cache_path)
-        return result_data
+        if 'error' in result_data:
+            self.log.error ("Error (%s) during running command %s: %r" %
```

Comment:
This will again show in sentry as an error during the ingestion flow. if we are planning to by pass this, i think we should change it to info